tests/test_main_run_main.py

```
@@ -0,0 +1,481 @@
+    assert result == 0
+    mock_print_banner.assert_called_once()
+    mock_load_config.assert_called_once_with(args=mock_args)
+    mock_asyncio_run.assert_called_once()
+
+
+@patch("asyncio.run")
+@patch("mmrelay.config.set_config")
+@patch("mmrelay.config.load_config")
+@patch("mmrelay.main.print_banner")
+def test_run_main_missing_config_keys(
+    mock_print_banner, mock_load_config, mock_set_config, mock_asyncio_run, mock_config
```

Ruff (0.15.12)

[warning] Unused function argument: `mock_print_banner`

(ARG001)